daml-script/dump/src/main/scala/com/daml/script/dump/Encode.scala

```diff
@@ -335,6 +333,15 @@ private[dump] object Encode {
 
   private def encodeSelector(selector: Selector): Doc = Doc.str(selector.i)
 
+  private def submit(partyMap: Map[Party, String], submitters: Set[Party], isTree: Boolean): Doc = {
```

Comment:
I'm not too happy with this `isTree` parameter. However, for now it seems like an improvement over the previous state.